client/state/data-getters/index.js

```diff
@@ -209,3 +209,62 @@ export const requestSiteAlerts = siteId => {
 		}
 	);
 };
+
+export const requestAtomicSFTPDetails = siteId =>
+	requestHttpData(
+		`atomic-hosting-data-${ siteId }`,
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/hosting/ssh`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			freshness: 5 * 60 * 1000,
+			fromApi: () => ( { username } ) => {
+				return [ `atomic-hosting-data-${ siteId }`, { username } ];
+			},
+		}
+	);
+
+export const resetAtomicSFTPUserPassword = siteId =>
```

Comment:
Going to refactor these, but also feel like these should be located somewhere else (like in the data-layer directory) _but_ i don't really see examples of usages of `reuqestHttpData` in there. 🤷‍♀

Reply:
for now it's fine to leave them here. we don't have much in the way of `POST` examples and this is as good of a place as any until we have a better handle on what to do